wallets/provider-all/src/index.ts

```diff
@@ -59,5 +60,6 @@ export const allProviders = (enviroments?: Enviroments) => {
     frontier,
     taho,
     braavos,
+    solflareSnap,
```

Comment:
Please change the code, so I could see solana snap in connect wallet right after the metamask icon.